services/movie_session.py

```diff
@@ -45,10 +45,8 @@ def delete_movie_session_by_id(session_id: int) -> None:
 
 
 def get_taken_seats(movie_session_id: int) -> list[dict]:
-    movie_session = MovieSession.objects.get(id=movie_session_id)
-    taken_seats = list(
-        movie_session.tickets.order_by(
-            "row", "seat"
-        ).values("row", "seat")
+    return list(
+        Ticket.objects.filter(movie_session_id=movie_session_id)
+        .order_by("row", "seat")
```

Comment:
why do u need order_by here?

Reply:
To have sorting to only by row, but also by seat as in example of README.md)